rustsec/src/osv/advisory.rs

```diff
@@ -150,6 +151,13 @@ pub struct OsvDatabaseSpecific {
     informational: Option<Informational>,
 }
 
+#[derive(Debug, Clone, Serialize)]
+pub struct MainOsvDatabaseSpecific {
+    license: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    attribution_url: Option<Url>,
```

Comment:
I'd prefer not to have a separate `attribution_url` field in OSV. 

Since this is a database-specific field, licence compliance would require special handling for RustSec in case the attribution URL is not also duplicated in the `references` field.

I'd rather put the attribution URL as the first link in references. Same logic as the `url` field - all reasonable users are automatically compliant.

Reply:
This was because of the field in the advisory metadata, once we decide to use `url` it makes no sense anymore.